### ISSUE TITLE
Make developer tests more robust

### ISSUE THREAD

**Issue by bitcoinhodler** [GlacierProtocol/GlacierProtocol#26]
The developer tests for GlacierScript (#24) should be more robust. @gracenoah ran into a couple of install hiccups that we may want to document, or fail fast in the tests for:

- check bitcoind is installed in the expected location (the error we get back is that bitcoind fails to start, even if it isn't installed on the host at all)
- check zbar-tools (for zbarimg) and qrencode are installed
- I'm not sure which version / toolchain of pgrep you're using but the one I have installed doesn't have the same options:

```
pgrep: invalid option -- '-'
Usage: pgrep [-flvx] [-d DELIM] [-n|-o] [-P PPIDLIST] [-g PGRPLIST] [-s SIDLIST]
        [-u EUIDLIST] [-U UIDLIST] [-G GIDLIST] [-t TERMLIST] [PATTERN]

```

**Comment by bitcoinhodler** [comment]
@gracenoah can you tell me what changes you made to get pgrep to work for you?

**Comment by gracenoah** [comment]
@bitcoinhodler: I replaced `--full` with `-f`, diff here:

```
diff --git Makefile Makefile
index b778015..a3c9376 100644
--- Makefile
+++ Makefile
@@ -42,11 +42,11 @@ $(foreach t,$(all-tests),$(eval $(call set_compteur, $(t))))
 define cleanup_bitcoind =
 @mkdir -p $(RUNDIR)/bitcoin-test-data
 @bitcoin-cli -testnet -rpcport=$(compteur) -datadir=$(RUNDIR)/bitcoin-test-data stop >/dev/null 2>&1 || exit 0
-@if pgrep --full "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then sleep 1; fi
-@if pgrep --full "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then sleep 1; fi
-@if pgrep --full "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then sleep 1; fi
-@if pgrep --full "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then sleep 1; fi
-@if pgrep --full "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then echo Error: unable to stop bitcoind on port $(compteur); exit 1; fi
+@if pgrep -f "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then sleep 1; fi
+@if pgrep -f "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then sleep 1; fi
+@if pgrep -f "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then sleep 1; fi
+@if pgrep -f "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then sleep 1; fi
+@if pgrep -f "^bitcoind -testnet -rpcport=$(compteur)" >/dev/null; then echo Error: unable to stop bitcoind on port $(compteur); exit 1; fi
 @sleep 1
 @rm -rf $(RUNDIR)/bitcoin-test-data
 endef
```

**Comment by bitcoinhodler** [comment]
Fixed by PR #29
